tests/js/yaml.tests.ts

```diff
@@ -0,0 +1,89 @@
+import { expect } from "chai";
+import fs from "fs";
+import path from "path";
+
+import { readYAMLFile, writeYAMLFile } from "../../src/js/server/yaml";
+import { expandTemplate } from "../../src/js/shared/str";
```

Comment:
There should be a separate section for JSON <> YAML conversion (without filesystem)